test/test_game.py

```diff
@@ -838,6 +838,32 @@ def test_penalization_after_invalid_moves(
         self.assertEqual(game.current_player.score, expected_score)
         self.assertEqual(game.game_is_active, game_active)
 
+    @parameterized.expand([
+        (PLAYER_1, 10, PLAYER_2, 9),
+        (PLAYER_1, 200, PLAYER_2, 199),
+        (PLAYER_2, 10, PLAYER_1, 9),
+        (PLAYER_2, 15, PLAYER_1, 14),
+    ])
+    def test_next_turn(self, current_player, current_remaining_moves,
+                       expected_player, expected_remainig_moves):
+
+        game = WumpusGame()
+
+        if PLAYER_1 == current_player:
+            game.current_player = game.player_1
+        else:
+            game.current_player = game.player_2
+
+        game.remaining_moves = current_remaining_moves
+
+        game.next_turn()
+
+        player = game.current_player
+        remaining_moves = game.remaining_moves
```

Comment:
similarly to above I would use `actual_player` and `actual_remaining_moves` here.

I think it's easier to understand the comparison `actual_player == expected_player` vs having a `current_player` that is not _current_ anymore and the current one is simply called `player`

Reply:
Exellent i'll change it